From: Anthony Sottile <asottile@umich.edu>
Date: Wed, 27 Jun 2018 21:25:05 -0700
Subject: Work with older sphinx

---
 Doc/tools/extensions/pyspecific.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Doc/tools/extensions/pyspecific.py b/Doc/tools/extensions/pyspecific.py
index 70bdd17..e7b8a94 100644
--- a/Doc/tools/extensions/pyspecific.py
+++ b/Doc/tools/extensions/pyspecific.py
@@ -319,7 +319,7 @@ class PydocTopicsBuilder(Builder):
             status_iterator = self.status_iterator
 
         writer = TextWriter(self)
-        for label in status_iterator(pydoc_topic_labels,
+        for label in self.status_iterator(pydoc_topic_labels,
                                      'building topics... ',
                                      length=len(pydoc_topic_labels)):
             if label not in self.env.domaindata['std']['labels']:
